free-programming-books-fr.md

```diff
@@ -128,6 +128,8 @@
 
 ### Java
 
+* [Apprenez à programmer en Java](https://openclassrooms.com/courses/apprenez-a-programmer-en-java) -  Cyrille Herby
+* [Java et la programmation réseau](https://openclassrooms.com/courses/java-et-la-programmation-reseau) - Cyrille Herby
```

Comment:
These two are not really free. It keeps asking me for a premium account or something. Please remove them.